common/dtos.go

```diff
@@ -52,3 +54,18 @@ type InitialAccountAPI struct {
 	StakingValue string            `json:"stakingvalue"`
 	Delegation   DelegationDataAPI `json:"delegation"`
 }
+
+type AlteredAccountTokenAPIResponse struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+type AlteredAccountAPIResponse struct {
+	Address string                      `json:"address"`
+	Balance string                      `json:"balance"`
+	Tokens  []*outport.AccountTokenData `json:"tokens,omitempty"`
```

Comment:
At this point, I'm not sure whether we should use the `outport.*` structure or define a separate one. Can be left as it is, perhaps.

Reply:
added a todo to use `outport.AlteredAccount` instead of `AlteredAccountAPIResponse`

Reply:
used altered account struct from core